<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelegatedStaking</class>
 <widget class="QWidget" name="DelegatedStaking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>800</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>60</number>
   </property>
   <property name="bottomMargin">
    <number>45</number>
   </property>
   <item>
    <widget class="QLabel" name="labelExplanation">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create &lt;span style=&quot; font-weight:600;&quot;&gt;LPoS smart-contracts&lt;/span&gt; straight from your wallet. (&lt;span style=&quot; font-weight:600;&quot;&gt;Tip&lt;/span&gt;: This page works with coin-control)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Create Contract</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>-1</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="spacing">
               <number>20</number>
              </property>
              <property name="topMargin">
               <number>12</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>12</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Lease to:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Amount:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Fee Percent (%):</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Reward Address:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <property name="spacing">
               <number>20</number>
              </property>
              <property name="topMargin">
               <number>12</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>12</number>
              </property>
              <item>
               <widget class="QLineEdit" name="delegateTo">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>e.g. NGshe11sePfsXGNn6oJQ1CzeGMGhhYTQxV</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="BitcoinAmountField" name="payAmount">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="feePercent">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>e.g. 11.9 (11.9%)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="rewardTo">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>e.g. NRahe11sePfsXGNn6oJQ1CzeGMGhhYTQxV</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="spacing">
               <number>20</number>
              </property>
              <property name="topMargin">
               <number>6</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLineEdit" name="contractLabel">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>Contract label (optional)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkboxSubtractFeeFromAmount">
                <property name="text">
                 <string>Subtract fee from amount</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="enableFeePayout">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Enable fee payout</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sendButton">
                <property name="toolTip">
                 <string>Convert Ghosted NIX</string>
                </property>
                <property name="text">
                 <string>&amp;Send NIX</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Active Contracts</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="activeContractsView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>110</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>25</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>My Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Lease Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Contract Fee (%)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Fee Reward Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Amount in Contract</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BitcoinAmountField</class>
   <extends>QLineEdit</extends>
   <header>qt/nixamountfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
